contracts/libraries/TwabLib.sol

```diff
@@ -247,8 +247,20 @@ library TwabLib {
         return differenceInAmount / differenceInTime;
     }
 
-    /// @notice Calculates the TWAB for a given timestamp.  It interpolates as necessary.
-    /// @param _twabs The TWAB history
+    /** @notice Calculates a user TWAB for a target timestamp using the historical TWAB list.
```

Comment:
I would add that it linearly interpolates a twab using the TWAB history.

Reply:
Yes sir.